plugin/pulp/plugin/task_system.py

```diff
@@ -0,0 +1,34 @@
+from pulp.platform.models import Task
+from pulp.tasking.system import get_current_task_id
+
+
+class TaskController(object):
```

Comment:
Few things:
- Naming the module `plugin/pulp/plugin/tasking.py` is more consistent with `pulp.tasking`.  Also, the `_system` seems redundant.
- Having a controller outside of the django `app` seems odd. 
- Having a *controller* that manipulates a single model object seems inappropriate.
- The `_and_save_` part of the method name seems odd.  Why update the task and not save it?
- Why not add `Task.append_non_fatal_error()`? and include the `Task` model in the *plugin* package?